prime-router/src/main/kotlin/tokens/OktaAuthentication.kt

```diff
@@ -99,9 +73,72 @@ class OktaAuthentication(private val minimumLevel: PrincipalLevel = PrincipalLev
             // Perform authentication.  Throws exception if authentication fails.
             return jwtVerifier.decode(accessToken)
         }
+
+        /**
```

Comment:
This function is still widely used, for example by Settings API and LookupTable API and others.

However, once this PR is in, we can begin to migrate those over to the dual-authentication and authorization functions found in AuthenticatedClaims.kt, rather than this Okta-specific code.